### ISSUE TITLE
none of the custom prefix is working

### ISSUE THREAD

**Issue by huyhoang160593** [dejmedus/tailwind-sorter#9]
When I setting up the prefix "cn(" in the setting, they don't seem to work in the code and not sorting the classes. Things are fine if I use only string in className but with any of the prefix, they are not working.

- Setting in VSCode
```json
  "tailwindSorter.customPrefixes": [
    "twMerge(",
    "clsx(",
    "cva(",
    "cn("
  ],
```
- VSCode version: 1.92.0
- Operating Systems: Window 11 Version 23H2

**Comment by dejmedus** [comment]
Thanks for catching this! 

It seems the regex wasn't happy with brackets or spaces before/after the prefix, for example in: 
`className={twMerge(  'flex p-4')}>`

You can check the [added tests](https://github.com/dejmedus/tailwind-sorter/pull/10/commits/03be020f8c2e852713b4fd8c1ef4f0034205f917) to see if they cover your use case.

Should be fixed in version 0.0.81, but if not, please let me know.

**Comment by huyhoang160593** [comment]
> Thanks for catching this!
> 
> It seems the regex wasn't happy with brackets or spaces before/after the prefix, for example in: `className={twMerge( 'flex p-4')}>`
> 
> You can check the [added tests](https://github.com/dejmedus/tailwind-sorter/pull/10/commits/03be020f8c2e852713b4fd8c1ef4f0034205f917) to see if they cover your use case.
> 
> Should be fixed in version 0.0.81, but if not, please let me know.

@dejmedus thanks you so much for the fast fix, this is now working fine with my settings
